lib/mongo.js

```diff
@@ -0,0 +1,125 @@
+const mongo = require('./mongo_conn');
+const { ObjectID } = require('mongodb');
+const CONFIG = mongo.config;
+const LIMIT = 20;
+
+class Mongo {
+
+    constructor(table, dbName = CONFIG.AMK_MONGO_DEFAULT_DATABASE) {
+        this.TABLE = table;
+        this.client = mongo();
+        this.dbName = dbName;
+    }
+
+    load() {
+        return this.client.then((client) => {
+            return client.db(this.dbName)
+                .collection(this.TABLE);
+        });
```

Comment:
Do you also consider to use `async`, `await` instead of promise?

Reply:
no. In fact, async also returns a promise